src/util/Constants.js

```diff
@@ -764,6 +764,7 @@ exports.APIErrors = {
   ANNOUNCEMENT_EDIT_LIMIT_EXCEEDED: 20022,
   CHANNEL_HIT_WRITE_RATELIMIT: 20028,
   CONTENT_NOT_ALLOWED: 20031,
+  GUILD_PREMIUM_LEVEL_TOO_LOW: 20035,
```

Comment:
| ⚠️ Pending on https://github.com/discord/discord-api-docs/pull/3189 to be merged before we can include this change in master. |
| --- |

Reply:
Upstream PR landed! 🎉